client/app/pages/queries-list/index.js

```diff
@@ -8,9 +8,13 @@ import './queries-list.css';
 class QueriesListCtrl {
   constructor($scope, $location, Events, Query, currentUser) {
     const page = parseInt($location.search().page || 1, 10);
+
     const orderSeparator = '-';
-    const pageOrder = $location.search().order || 'created_at';
-    const pageOrderReverse = pageOrder.startsWith(orderSeparator);
+    this.pageOrder = $location.search().order || '-created_at';
+    this.pageOrderReverse = this.pageOrder.startsWith(orderSeparator);
+    if (this.pageOrderReverse) {
+      this.pageOrder = this.pageOrder.substr(1);
+    }
```

Comment:
This basically makes sure that thie orderByField parameter that is passed to the LivePaginator is always a value without the prefix `-` and allow the sort icon to render properly. Before this we've basically relied on implicit behavior of the ascending ordering that doesn't have a prefix.